yarn.lock

```diff
@@ -2062,7 +2062,15 @@
   resolved "https://registry.yarnpkg.com/@types/q/-/q-1.5.5.tgz#75a2a8e7d8ab4b230414505d92335d1dcb53a6df"
   integrity sha512-L28j2FcJfSZOnL1WBjDYp2vUHCeIFlyYI/53EwD/rKUBQ7MtUUfbQWiyKJGpcnv4/WgrhWsFKrcPstcAt/J0tQ==
 
-"@types/react-native@~0.70.6":
+"@types/react-native-vector-icons@^6.4.6":
```

Comment:
You can get rid of this package too since we aren't using `react-native-elements`. -- `yarn remove react-native-vector-icons`